src/main/java/com/configcat/ConfigCatClient.java

```diff
@@ -452,25 +452,25 @@ private <T> T getValueFromSettingsMap(Class<T> classOfT, SettingResult settingRe
         try {
             Map<String, Setting> settings = settingResult.settings();
             if (settings.isEmpty()) {
-                String errorMessage = "Config JSON is not present. Returning defaultValue: [" + defaultValue + "].";
-                this.logger.error(errorMessage);
+                String errorMessage = "Config JSON is not present. Returning the `defaultValue` parameter that you specified in your application: '" + defaultValue + "'.";
+                this.logger.error("[1000] " + errorMessage);
                 this.configCatHooks.invokeOnFlagEvaluated(EvaluationDetails.fromError(key, defaultValue, errorMessage, user));
                 return defaultValue;
             }
 
             Setting setting = settings.get(key);
             if (setting == null) {
-                String errorMessage = "Value not found for key " + key + ". Here are the available keys: " + String.join(", ", settings.keySet());
-                this.logger.error(errorMessage);
+                String errorMessage = "Failed to evaluate setting '" + key + "' (the key was not found in config JSON). Returning the `defaultValue` parameter that you specified in your application: '" + defaultValue + "'. Available keys: " + String.join(", ", settings.keySet()) + ".";
```

Comment:
Avaliable keys should be surrounded by single quotes like `'Key1', 'Key2', 'Key3'`

Reply:
Fixed.